test/suite/test_compact07.py

```diff
@@ -143,9 +136,6 @@ def test_compact07(self):
         # Check that foreground compaction has done some work on the small table.
         self.assertGreater(self.get_pages_rewritten(uri_small), 0)
 
-        # Check that we have some files in the background compaction tracking list.
```

Comment:
I think we should remove this as the stat may be back to 0 by now if background compaction has compacted everything and there is no more work to do by the time we get here?

Reply:
Or I could move it earlier, the debug idle time is actually 10 seconds:
```
WT_BACKGROUND_COMPACT_MAX_IDLE_TIME_DEBUG
```

Reply:
happy to remove it, it's flakey with timing.

Reply:
I moved it to earlier in the test
8fea41e78b08af9cb435e5234c3d9b5a2c5abc9b